### PR TITLE
fix golint & ineffassign failures

### PR DESCRIPTION
**Pull request by FayerZhang**

### CODE REVIEW

File: boskos/ranch/ranch_test.go [hunk 1]
```diff
@@ -1150,7 +1150,7 @@ func TestSyncResources(t *testing.T) {
 		if !reflect.DeepEqual(resources, tc.expectedRes) {
 			t.Errorf("Test %v: \n got \t\t%v, \n expected \t%v", tc.name, resources, tc.expectedRes)
 		}
-		lfs, err := c.Storage.GetDynamicResourceLifeCycles()
```
Comment (fejta) [inline boskos/ranch/ranch_test.go]: handle the error please?

Comment (FayerZhang) [reply boskos/ranch/ranch_test.go]: I've done

File: boskos/ranch/ranch_test.go [hunk 2]
```diff
@@ -1151,6 +1151,10 @@ func TestSyncResources(t *testing.T) {
 			t.Errorf("Test %v: \n got \t\t%v, \n expected \t%v", tc.name, resources, tc.expectedRes)
 		}
 		lfs, err := c.Storage.GetDynamicResourceLifeCycles()
+		if err != nil {
+			t.Errorf("failed to get dynamic resources life cycles")
```
Comment (fejta) [inline boskos/ranch/ranch_test.go]: Please also include err in the error message

File: prow/repoowners/repoowners_test.go [hunk 3]
```diff
@@ -1265,6 +1265,9 @@ reviewers:
 			t.Errorf("result '%s' is differ from expected: '%s'", s, test.expected)
 		}
 		simple, err := LoadSimpleConfig(b)
+		if err != nil {
+			t.Errorf("unexpected error when load simple config")
```
Comment (fejta) [inline prow/repoowners/repoowners_test.go]: Same thing in this file (include the error message)

File: prow/repoowners/repoowners_test.go [hunk 4]
```diff
@@ -1323,7 +1326,10 @@ options: {}
 		if test.expected != s {
 			t.Errorf("result '%s' is differ from expected: '%s'", s, test.expected)
 		}
-		full, err := LoadFullConfig(b)
+		full, _ := LoadFullConfig(b)
```
Comment (fejta) [inline prow/repoowners/repoowners_test.go]: err not _

File: boskos/ranch/ranch_test.go (hunk 2 above)
Comment (FayerZhang) [reply boskos/ranch/ranch_test.go]: I have done

File: prow/repoowners/repoowners_test.go (hunk 3 above)
Comment (FayerZhang) [reply prow/repoowners/repoowners_test.go]: I have done

File: prow/repoowners/repoowners_test.go (hunk 4 above)
Comment (FayerZhang) [reply prow/repoowners/repoowners_test.go]: I have done

File: prow/plugins/retitle/retitle.go [hunk 5]
```diff
@@ -135,13 +135,12 @@ func handleGenericComment(gc githubClient, isTrusted func(string) (bool, error),
...
-		issue, err := gc.GetIssue(org, repo, number)
-		if err != nil {
-			return err
-		}
-		issue.Title = newTitle
-		_, err = gc.EditIssue(org, repo, number, issue)
+	}
+	issue, err := gc.GetIssue(org, repo, number)
+	if err != nil {
 		return err
```
Comment (fejta) [inline prow/plugins/retitle/retitle.go]: fyi: in general I think it is better form when a function can return a variety of errors to help disambiguate between them, aka: return `fmt.Errorf("get issue: %v", err")` vs `fmt.Errorf("edit issue: %v", err)` later

But nothing in this file does that yet so makes sense to follow existing convention (but would make a good follow up PR if you're interested)

**Pull request merged**